srcpkgs/kodi/template

```diff
@@ -106,15 +114,41 @@ pre_configure() {
 
 	# Build tools needed to run on the host
 	if [ "$CROSS_BUILD" ]; then
+		cd tools/depends && ./bootstrap
+		sed -e "s|uclibc|musl|g" -i configure
```

Comment:
vsed

Reply:
sure i can, what's that catch if vsed detects no change? just to remove it in the future?

Reply:
yes